### PR TITLE
[139] Product Page using HTML

### PR DESCRIPTION
**Pull request by titilayo20**
I designed a product page for my E-book website using HTML. Below are the screenshots of HTML elements implemented:

IMG_20201130_123538
IMG_20201130_123610
IMG_20201130_123625
IMG_20201130_123631

### CODE REVIEW

**Comment by Ireti203** [comment]
You should attach screenshots of your view i.e the webpage and not your code.

File: submissions/titilayoOlutola/HTML/Html task.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>FindMeABook</title>
+    </head>
+    <body>
+        <h1>FindMeABook</h1>
+
```
Comment (Ireti203) [inline submissions/titilayoOlutola/HTML/Html task.html:9]: Remove whitespace.

File: submissions/titilayoOlutola/HTML/Html task.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                <td><a href="#">Educational</a></td>
+                <td><a href="#">Autobiographies</a></td>
+                <td><a href="#">Historicals</a></td>
+                <td><a href="#">Others</a></td>
+            </tr>
+        </tables>
+    </body>
+    <section>
+        <h2>A Gentleman In Moscow By Amor Towles</h2>
+
```
Comment (Ireti203) [inline submissions/titilayoOlutola/HTML/Html task.html:25]: Remove whitespace

File: submissions/titilayoOlutola/HTML/Html task.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    <hr>
+    <section>
+        <h3>Description</h3>
+        <p><strong>File size: 10MB</strong></p>
+        <p><strong>
+            Pages: 462
+        </strong></p>
+        <p><strong>
+            Synopsis: With his breakout debut novel, Rules of Civility, Amor Towles established himself as a master of absorbing, sophisticated fiction, bringing late 1930s Manhattan to life with splendid atmosphere and a flawless command of style. Readers and critics were enchanted; as NPR commented, “Towles writes with grace and verve about the mores and manners of a society on the cusp of radical change.”
+
```
Comment (Ireti203) [inline submissions/titilayoOlutola/HTML/Html task.html:40]: Remove whitespace. Also line 39 IS too long. Use the enter key to break it into the next line.

File: submissions/titilayoOlutola/HTML/Html task.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+            A Gentleman in Moscow immerses us in another elegantly drawn era with the story of Count Alexander Rostov. When, in 1922, he is deemed an unrepentant aristocrat by a Bolshevik tribunal, the count is sentenced to house arrest in the Metropol, a grand hotel across the street from the Kremlin. Rostov, an indomitable man of erudition and wit, has never worked a day in his life, and must now live in an attic room while some of the most tumultuous decades in Russian history are unfolding outside the hotel’s doors. Unexpectedly, his reduced circumstances provide him a doorway into a much larger world of emotional discovery.
```
Comment (Ireti203) [inline submissions/titilayoOlutola/HTML/Html task.html:41]: Same here

File: submissions/titilayoOlutola/HTML/Html task.html [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
```
Comment (Ireti203) [inline submissions/titilayoOlutola/HTML/Html task.html:42]: Remove whitespace

File: submissions/titilayoOlutola/HTML/Html task.html [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+            A Gentleman in Moscow immerses us in another elegantly drawn era with the story of Count Alexander Rostov. When, in 1922, he is deemed an unrepentant aristocrat by a Bolshevik tribunal, the count is sentenced to house arrest in the Metropol, a grand hotel across the street from the Kremlin. Rostov, an indomitable man of erudition and wit, has never worked a day in his life, and must now live in an attic room while some of the most tumultuous decades in Russian history are unfolding outside the hotel’s doors. Unexpectedly, his reduced circumstances provide him a doorway into a much larger world of emotional discovery.
+            
+            Brimming with humour, a glittering cast of characters, and one beautifully rendered scene after another, this singular novel casts a spell as it relates the count’s endeavour to gain a deeper understanding of what it means to be a man of purpose.
```
Comment (Ireti203) [inline submissions/titilayoOlutola/HTML/Html task.html:43]: Same here

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/titilayoOlutola/HTML/Html task.html (hunk 1 above)
Comment (titilayo20) [reply submissions/titilayoOlutola/HTML/Html task.html]: done

File: submissions/titilayoOlutola/HTML/Html task.html (hunk 3 above)
Comment (titilayo20) [reply submissions/titilayoOlutola/HTML/Html task.html]: done

**Comment by titilayo20** [comment]
Thanks for the review, I have made the required changes.  
 
  On Mon, Nov 30, 2020 at 11:47, Ireti203<notifications@github.com> wrote:   


@Ireti203 requested changes on this pull request.

In submissions/titilayoOlutola/HTML/Html task.html:
> @@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>FindMeABook</title>
+    </head>
+    <body>
+        <h1>FindMeABook</h1>
+

Remove whitespace.

In submissions/titilayoOlutola/HTML/Html task.html:
> +        <h4> <em>your-library-on-the-go</em></h4>
+        <hr>
+        <tables>
+            <tr>
+                <td><a href="#">Fiction</a></td>
+                <td><a href="#">Non-Fiction</a></td>
+                <td><a href="#">Educational</a></td>
+                <td><a href="#">Autobiographies</a></td>
+                <td><a href="#">Historicals</a></td>
+                <td><a href="#">Others</a></td>
+            </tr>
+        </tables>
+    </body>
+    <section>
+        <h2>A Gentleman In Moscow By Amor Towles</h2>
+

Remove whitespace

In submissions/titilayoOlutola/HTML/Html task.html:
> +
+        
+        
+        <h3><a href="#">DOWNLOAD</a></h3>
+        <h3><a href="#">ADD TO LIBRARY</a></h3>
+    </section>
+    <hr>
+    <section>
+        <h3>Description</h3>
+        <p><strong>File size: 10MB</strong></p>
+        <p><strong>
+            Pages: 462
+        </strong></p>
+        <p><strong>
+            Synopsis: With his breakout debut novel, Rules of Civility, Amor Towles established himself as a master of absorbing, sophisticated fiction, bringing late 1930s Manhattan to life with splendid atmosphere and a flawless command of style. Readers and critics were enchanted; as NPR commented, “Towles writes with grace and verve about the mores and manners of a society on the cusp of radical change.”
+

Remove whitespace. Also line 39 IS too long. Use the enter key to break it into the next line.

In submissions/titilayoOlutola/HTML/Html task.html:
> +        
+        
+        <h3><a href="#">DOWNLOAD</a></h3>
+        <h3><a href="#">ADD TO LIBRARY</a></h3>
+    </section>
+    <hr>
+    <section>
+        <h3>Description</h3>
+        <p><strong>File size: 10MB</strong></p>
+        <p><strong>
+            Pages: 462
+        </strong></p>
+        <p><strong>
+            Synopsis: With his breakout debut novel, Rules of Civility, Amor Towles established himself as a master of absorbing, sophisticated fiction, bringing late 1930s Manhattan to life with splendid atmosphere and a flawless command of style. Readers and critics were enchanted; as NPR commented, “Towles writes with grace and verve about the mores and manners of a society on the cusp of radical change.”
+
+            A Gentleman in Moscow immerses us in another elegantly drawn era with the story of Count Alexander Rostov. When, in 1922, he is deemed an unrepentant aristocrat by a Bolshevik tribunal, the count is sentenced to house arrest in the Metropol, a grand hotel across the street from the Kremlin. Rostov, an indomitable man of erudition and wit, has never worked a day in his life, and must now live in an attic room while some of the most tumultuous decades in Russian history are unfolding outside the hotel’s doors. Unexpectedly, his reduced circumstances provide him a doorway into a much larger world of emotional discovery.

Same here

In submissions/titilayoOlutola/HTML/Html task.html:
> +        
+        <h3><a href="#">DOWNLOAD</a></h3>
+        <h3><a href="#">ADD TO LIBRARY</a></h3>
+    </section>
+    <hr>
+    <section>
+        <h3>Description</h3>
+        <p><strong>File size: 10MB</strong></p>
+        <p><strong>
+            Pages: 462
+        </strong></p>
+        <p><strong>
+            Synopsis: With his breakout debut novel, Rules of Civility, Amor Towles established himself as a master of absorbing, sophisticated fiction, bringing late 1930s Manhattan to life with splendid atmosphere and a flawless command of style. Readers and critics were enchanted; as NPR commented, “Towles writes with grace and verve about the mores and manners of a society on the cusp of radical change.”
+
+            A Gentleman in Moscow immerses us in another elegantly drawn era with the story of Count Alexander Rostov. When, in 1922, he is deemed an unrepentant aristocrat by a Bolshevik tribunal, the count is sentenced to house arrest in the Metropol, a grand hotel across the street from the Kremlin. Rostov, an indomitable man of erudition and wit, has never worked a day in his life, and must now live in an attic room while some of the most tumultuous decades in Russian history are unfolding outside the hotel’s doors. Unexpectedly, his reduced circumstances provide him a doorway into a much larger world of emotional discovery.
+            

Remove whitespace

In submissions/titilayoOlutola/HTML/Html task.html:
> +        <h3><a href="#">DOWNLOAD</a></h3>
+        <h3><a href="#">ADD TO LIBRARY</a></h3>
+    </section>
+    <hr>
+    <section>
+        <h3>Description</h3>
+        <p><strong>File size: 10MB</strong></p>
+        <p><strong>
+            Pages: 462
+        </strong></p>
+        <p><strong>
+            Synopsis: With his breakout debut novel, Rules of Civility, Amor Towles established himself as a master of absorbing, sophisticated fiction, bringing late 1930s Manhattan to life with splendid atmosphere and a flawless command of style. Readers and critics were enchanted; as NPR commented, “Towles writes with grace and verve about the mores and manners of a society on the cusp of radical change.”
+
+            A Gentleman in Moscow immerses us in another elegantly drawn era with the story of Count Alexander Rostov. When, in 1922, he is deemed an unrepentant aristocrat by a Bolshevik tribunal, the count is sentenced to house arrest in the Metropol, a grand hotel across the street from the Kremlin. Rostov, an indomitable man of erudition and wit, has never worked a day in his life, and must now live in an attic room while some of the most tumultuous decades in Russian history are unfolding outside the hotel’s doors. Unexpectedly, his reduced circumstances provide him a doorway into a much larger world of emotional discovery.
+            
+            Brimming with humour, a glittering cast of characters, and one beautifully rendered scene after another, this singular novel casts a spell as it relates the count’s endeavour to gain a deeper understanding of what it means to be a man of purpose.

Same here

**Comment by Ireti203** [comment]
There should be no spacing in the naming of files. Kindly correct your naming...you could use camel-casing.

**Comment by titilayo20** [comment]
I don't understand which files I'm to use camel-case for. All the book titles?  
 
  On Mon, Nov 30, 2020 at 14:12, Ireti203<notifications@github.com> wrote:   


There should be no spacing in the naming of files. Kindly correct your naming...you could use camel-casing.

**Comment by Ireti203** [comment]
> I don't understand which files I'm to use camel-case for. All the book titles?  On Mon, Nov 30, 2020 at 14:12, Ireti203<notifications@github.com> wrote: There should be no spacing in the naming of files. Kindly correct your naming...you could use camel-casing. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe

Your html file is not properly named. You could use html-task.html instead. Spaces are not recommended in namings.

**Comment by titilayo20** [comment]
I've renamed it, thank you. 
 
  On Mon, Nov 30, 2020 at 16:14, Ireti203<notifications@github.com> wrote:   



I don't understand which files I'm to use camel-case for. All the book titles? On Mon, Nov 30, 2020 at 14:12, Ireti203notifications@github.com wrote: There should be no spacing in the naming of files. Kindly correct your naming...you could use camel-casing. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe


Your html file is not properly named. You could use html-task.html instead. Spaces are not recommended in namings.

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**